test/fixtures/addresses.yml

```diff
@@ -0,0 +1,13 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
dont add test files if not being used. Avoid using scaffold if you're using it

Reply:
Removed test file
I was using - rails g model, so it was generating test files as well